src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplGetCaseByIdTest.java

```diff
@@ -119,25 +125,94 @@ public void testGetCaseByCaseId_householdIndividualCase() throws CTPException {
     }
   }
 
+  @Test
+  public void testGetLatestFromCacheWhenResultsFromBothRmAndCache() throws Exception {
```

Comment:
I can't see why you've got 4 test methods. Aren't there just 2 circumstances? 
1. RM case newer than cached case.
2. Cached case newer than RM case

Reply:
I was re-using existing code that tests the getLatestByUprn() method by checking that it will still return the latest case even when the created/updated times are only 1 second apart.

Reply:
I have now added a test for case not found too by the way. And I've also added one to test that it will return the CachedCase if there are no RM cases found - but I can remove that one if it's overkill?